services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/AmenitiesTest.java

```diff
@@ -0,0 +1,32 @@
+package com.mapbox.api.directions.v5.models;
+
+import com.mapbox.core.TestUtils;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class AmenitiesTest extends TestUtils {
+
+  @Test
+  public void sanity(){
+    assertNotNull(getDefault());
+  }
+
+  private Amenities getDefault() {
+    return Amenities.builder()
+        .type("restaurant")
+        .name("starbucks")
+        .build();
+  }
+
+  @Test
+  public void testSerializableObject() throws Exception {
+    Amenities amenities = Amenities.builder()
+        .type("restaurant")
+        .name("starbucks")
+        .build();
+    byte[] serialized = TestUtils.serialize(amenities);
+    assertEquals(amenities, deserialize(serialized, Amenities.class));
```

Comment:
Could we structure the test following the Triple A?

Same thing applies to other tests below 👇

Reply:
Okay I forgot to remove the term `test` from a couple of tests. I have done that now. @Guardiola31337